_about/notes/0.8.md

```diff
@@ -8,4 +8,55 @@ type: markdown
 ---
 {% include home.html %}
 
-FILL IN BEFORE RELEASE
+In addition to the usual pile of bug fixes and performance improvements, this release includes the new or
+updated features detailed below.
+
+## Networking
+
+- **Revamped Traffic Management Model**. We're finally ready to take the wraps off our new traffic
+management configuration model. The new model adds many new features and addresses usability issues
+with the prior model. There is a conversion tool built into `istioctl` to help migrate your config from
+the old model. [Learn more about the new traffic management model]({{home}}/docs/tasks/traffic-management-v1alpha3/).
+
+- **Envoy V2**. Users can choose to inject Envoy V2 as the sidecar. In this mode, Pilot uses the V2 Envoy API to program the proxies
```

Comment:
Push new configuration to dataplane: By taking advantage of Envoy's new gRPC based apis (envoy-v2), we are now able to push new configurations to envoys in the data plane rather than frequent polling. The push model should reduce the CPU consumption on Pilot as well as Envoys. In addition, the new gRPC configuration in Envoy offers several scalability improvements.